### DISCUSSION TITLE
Replace Bullhorn Logo with Company logo

### DISCUSSION THREAD

**Discussion by ecoatROP** [bullhorn/career-portal#507] [General]
How do you change the "powered by Bullhorn" logo on the dashboard? I am trying to change it to our company's logo. 
![Bullhorn](https://user-images.githubusercontent.com/127762402/224730589-36e68806-f6c6-4ac9-a1c6-e44204aefc92.PNG)

**Comment by pbearne** [comment]
You will have to use CSS.
First, hide it and then add an :after with the image.

Checkout [Matador Jobs](https://matadorjobs.com/) for and upgrade to OSCP

**Reply by ecoatROP** [reply]
I have the bullhorn plug-in installed in wordpress for the site. Are you saying I need to add the Matador plug-in in order to use CSS and edit the image?

**Reply by pbearne** [reply]
No, I am suggesting that you might want to replace/upgrade the OSCP; read this whitepaper to understand why
https://matadorjobs.com/why-replace-oscp-with-matador-jobs/
You have hit one of the limitations that Matador Jobs doesn't have, as it is fully white-labelled

You can do what you want, but you have to fight OSCP and redo the changes every time you upgrade :-)

Reach out for a quick call if that will help
paul@matadorjobs.com
